R/constants.R

```diff
@@ -130,9 +130,11 @@ ELECTRICITY_INPUT_FUELS<- c( "biomass", "coal", "gas", "refined liquids" )
 # and should contain all historical years used by other modules
 socioeconomics.MADDISON_HISTORICAL_YEARS <- seq(1700, 1900, 50) # Years for which to use Maddison data
 socioeconomics.UN_HISTORICAL_YEARS <- c(1950, 1971:2010) # Years for which to use UN data
-
+socioeconomics.AGG_BASE_YEAR <- 1950 # Year used to generate ratios for region -> country downscaling
+socioeconomics.FINAL_HIST_YEAR <- 2010 # Final historical year
```

Comment:
Actually, I think we need to leave it as 2010 (see comment above) because we need to be able to match SSP and UN population data and the SSP data start in 2010. Unless there is ever a new SSP database (in which case this would likely need to be rewritten anyway), 2010 is always going to be this "historical" year for the SSPs (while 2015 is the first future year in the SSPs).